tests/wpt/mozilla/tests/mozilla/FileAPI/file-select.html

```diff
@@ -9,23 +9,21 @@
 <script>
 
 async_test(function() {
-  var e = document.getElementById("file-input");
+  var fileInput = document.getElementById("file-input");
+  assert_true(fileInput.selectFiles != undefined, "selectFiles is not defined")
 
-  assert_true(e.selectFiles != undefined, "selectFiles is not defined")
 
-  e.selectFiles(["./tests/wpt/mozilla/tests/mozilla/test.txt"]);
+  fileInput.addEventListener("change", () => {
+    assert_true(fileInput.files.length > 0, "test.txt is not selected");
 
-  assert_true(e.files.length > 0, "test.txt is not selected");
-
-  var reader = new FileReader;
-
-  reader.onloadend = this.step_func(function(evt) {
-    assert_equals(evt.target.result, "hello, servo\n");
-
-    this.done();
+    var reader = new FileReader;
+    reader.onloadend = this.step_func(function(evt) {
+      assert_equals(evt.target.result, "hello, servo\n");
+      this.done();
+    });
```

Comment:
Nit: can be a single invocation as well:

```suggestion
    reader.onloadend = this.step_func_done(function(evt) {
      assert_equals(evt.target.result, "hello, servo\n");
    });
```

Reply:
Okay. I've done this.